app/views/layouts/darkswarm.html.haml

```diff
@@ -40,7 +40,8 @@
     = render "layouts/bugsnag_js"
 
     - if Spree::Config.stripe_connect_enabled
-      %script{:src => "https://js.stripe.com/v3/", :type => "text/javascript"}
+      = render "shared/include_stripe_js"
```

Comment:
I think the if clause can go into the partial

Reply:
The partial is used in other places that don't have the conditional. I was wondering about this though, do we even need to load Stripe JS here in the darkswarm layout? I think we already load it again at checkout if a Stripe method is selected..?

Reply:
Stripe recommends to load the code on every page because it improves their fraud detection (and data collection).

Reply:
nice idea Maikel, let's open a separate issue for that? load stripe only on shopping pages if it is enabled in the instance and if the distributor has a stripe payment method :ok_hand:

Reply:
Last time I looked Stripe was 50% of the transactions in the UK and AU and 20% in France. It's a lot, but it would mean more than half of the shoppers would not get stripe js on their browser :ok_hand: